django/views/i18n.py

```diff
@@ -301,8 +308,8 @@ def json_catalog(request, domain='djangojs', packages=None):
     """
     Return the selected language catalog as a JSON object.
 
-    Receives the same parameters as javascript_catalog(), but returns
-    a response with a JSON object of the following format:
+    Receives the same parameters as javascript_catalog() and returns a response
```

Comment:
Oops, I shouldn't have edited the docstring for the deprecated function. Could you use this wording "and returns a response..." for JSONCatalog instead?